e2e/specs/add-appointments.spec.ts

```diff
@@ -0,0 +1,114 @@
+import { expect } from '@playwright/test';
+import { generateRandomPatient, deletePatient, type Patient } from '../commands';
+import { test } from '../core';
+import { AppointmentsPage } from '../pages';
+
+let patient: Patient;
+
+test.beforeEach(async ({ api }) => {
+  patient = await generateRandomPatient(api);
```

Comment:
Should we start patient visit before recording the appointment, @Vijaykv5 what do you think?

Reply:
Oops I missed that out. Yes that right!
Thank you @kdaud  
I've updated new changes

Reply:
Appointments _should_ be able to be scheduled without requiring an active visit. That's a kind of basic feature of an appointment scheduling application.